functions/invite/encrypt.py

```diff
@@ -1,13 +1,7 @@
 from __future__ import print_function, unicode_literals
 import base64
-import sys
 import pyaes.aes
 
-if sys.version_info < (3, 0):
```

Comment:
For future reference, you can use `from builtins import input` here instead of manually checking the version. [This site](http://python-future.org/compatible_idioms.html) is a great reference for writing Python 2-3 compatible code.